### PR TITLE
Fix subprocess.wait to take None as a timeout argument

### PR DESCRIPTION
**Pull request by youknowone**

### CODE REVIEW

File: Lib/subprocess.py [hunk 1]
```diff
@@ -230,7 +230,7 @@ def call(*popenargs, timeout=None, **kwargs):
     """
     with Popen(*popenargs, **kwargs) as p:
         try:
-            return p.wait(timeout=timeout)
+            return p.wait(timeout)
```
Comment (youknowone) [inline Lib/subprocess.py]: is there an easy to add keyword argument support?

Comment (coolreader18) [reply Lib/subprocess.py]: Other than a FromArgs derive on a struct, I don't think so.

File: vm/src/stdlib/subprocess.rs [hunk 2]
```diff
@@ -142,8 +142,8 @@ impl PopenRef {
         self.process.borrow().exit_status()
     }
 
-    fn wait(self, timeout: OptionalArg<u64>, vm: &VirtualMachine) -> PyResult<()> {
-        let timeout = match timeout.into_option() {
+    fn wait(self, timeout: OptionalArg<Option<u64>>, vm: &VirtualMachine) -> PyResult<()> {
```
Comment (coolreader18) [inline vm/src/stdlib/subprocess.rs]: Could you add a `PopenWaitArgs` struct so that the `timeout=` kwarg works?

**Review by coolreader18 — CHANGES_REQUESTED** [review]

File: vm/src/stdlib/subprocess.rs (hunk 2 above)
Comment (youknowone) [reply vm/src/stdlib/subprocess.rs]: done

**Review by coolreader18 — APPROVED** [review]

**Pull request merged**